cmd/create/create.go

```diff
@@ -115,6 +116,9 @@ func (o *Options) Run(cmd *cobra.Command, args []string, cloud *common.Cloud) er
 		return err
 	}
 
+	logrus.Infof("Using identifier %s", id.Long())
+	defer fmt.Println(id.Long())
```

Comment:
Do we need this `defer` here?

Reply:
It was just to make sure that the identifier is printed to `stdout` at the end, both in case of error and success. Seeing it on the top of the logs wasn't particularly pleasant for me.

My goal was:
1. Print the identifier to the **`stderr`** user-oriented logs in the beginning.
2. Print the identifier to the **`stdout`** unix-oriented logs at the end.

Reply:
If it's too much, we can just **(2)** in the beginning as you originally proposed.

Reply:
I see. Ok, let's keep it as is for now.